### DISCUSSION TITLE
Grid form 是否支持 Form store 和 codes 权限码指令

### DISCUSSION THREAD

**Discussion by dingdayu** [vbenjs/vue-vben-admin#5223] [Q&A]
在列表搜索表单场景，是否可以将表单值临时存储，在通过 Tab 或前进后退 页面切换时，切换回来时，表单值仍存在，并刷新表单；
第二个问题是，在上面的场景中，表单的 Filed 是否可以通过 codes 指令进行权限的控制显示？

附属： 第三个问题，VxeGridProps 中，也想通过 codes 进行控制某一列是否可以进行权限控制（同时允许visible: false和自定义列）。

**Comment by HoldEvan** [comment]
+1
